src/app/csc/csc.component.scss

```diff
@@ -8,3 +8,8 @@
     bottom: 0.2rem;
   }
 }
+
+// This is so that the last item in the list doesn't have extra padding
+td.mat-mdc-cell ul li:not(:last-child) {
```

Comment:
This is good.  I think it maybe should be added also to the `topic` based search because that will make the formatting between the two tables consistent (maybe even pull this style "up" so it's not repeated?)

Reply:
I moved this code out to the shared.scss file to be used by all of the table td elements.